specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json

```diff
@@ -930,4 +947,4 @@
       "description": "SKU of an Azure Firewall."
     }
   }
-}
+}
```

Comment:
You might be missing the EditorConfig plug-in for your IDE if it's removing the trailing newline

Reply:
installed the extension and restored trailing newlines.